arc/conformance/plugin/Dockerfile

```diff
@@ -0,0 +1,36 @@
+ARG STEP_CLI_VERSION=0.18.0
+ARG STEP_CLI_IMAGE=smallstep/step-cli:${STEP_CLI_VERSION}
+FROM $STEP_CLI_IMAGE as step-cli
+
+FROM k8s.gcr.io/build-image/debian-base:bullseye-v1.1.0
+ARG KUBE_VERSION=v1.21.2
+ARG TARGETARCH
+
+# install dependencies
+RUN apt-get update -y && apt-get upgrade -y
+RUN DEBIAN_FRONTEND=noninteractive TZ=America/Los_Angeles apt-get install python3-pip bash curl apt-transport-https lsb-release gnupg -y --allow-change-held-packages
+RUN python3 -m pip install junit_xml
```

Comment:
Every `RUN` command adds a new layer (ref: https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#minimize-the-number-of-layers). Should we consider chaining these commands?